### PR TITLE
WIREUP/LOG: improved keepalive logging

### PR DESCRIPTION
**Pull request by hoopoepg**
- added missing log entries for keepalive flags

### CODE REVIEW

**Comment by hoopoepg** [comment]
@brminich more comments?

File: src/tools/info/tl_info.c [hunk 1]
```diff
@@ -323,6 +325,12 @@ static void print_iface_info(uct_worker_h worker, uct_md_h md,
             if (iface_attr.cap.flags & UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE) {
                 strncat(buf, " peer failure,", sizeof(buf) - strlen(buf) - 1);
             }
+            if (iface_attr.cap.flags & UCT_IFACE_FLAG_EP_CHECK) {
+                strncat(buf, " endpoint check,", sizeof(buf) - strlen(buf) - 1);
+            }
+            if (iface_attr.cap.flags & UCT_IFACE_FLAG_EP_KEEPALIVE) {
+                strncat(buf, " endpoint keepalive,", sizeof(buf) - strlen(buf) - 1);
```
Comment (yosefe) [inline src/tools/info/tl_info.c:332]: maybe just "ep_check" and "keepalive"

**Comment by yosefe** [comment]
BTW, shall we show ep_check lanes in `ucx_info -u t -e` ?

File: src/tools/info/tl_info.c (hunk 1 above)
Comment (hoopoepg) [reply src/tools/info/tl_info.c]: updated

**Review by hoopoepg — COMMENTED** [review]
may be add command line arg to select peer-failure lanes?

File: src/tools/perf/lib/libperf.c [hunk 2]
```diff
@@ -81,7 +81,9 @@ static const char *perf_iface_ops[] = {
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_EAGER_SHORT)]  = "tag eager short",
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_EAGER_BCOPY)]  = "tag eager bcopy",
     [ucs_ilog2(UCT_IFACE_FLAG_TAG_EAGER_ZCOPY)]  = "tag eager zcopy",
-    [ucs_ilog2(UCT_IFACE_FLAG_TAG_RNDV_ZCOPY)]   = "tag rndv zcopy"
+    [ucs_ilog2(UCT_IFACE_FLAG_TAG_RNDV_ZCOPY)]   = "tag rndv zcopy",
+    [ucs_ilog2(UCT_IFACE_FLAG_EP_CHECK)]         = "ep check",
+    [ucs_ilog2(UCT_IFACE_FLAG_EP_KEEPALIVE)]     = "ep keepalive"
```
Comment (brminich) [inline src/tools/perf/lib/libperf.c:86]: this seems to be excessive, what is the scenario when it will be used?

Comment (brminich) [reply src/tools/perf/lib/libperf.c:86]: but we will not request `UCT_IFACE_FLAG_EP_CHECK` or `UCT_IFACE_FLAG_EP_CHECK` flags explicitly.

Comment (hoopoepg) [reply src/tools/perf/lib/libperf.c:86]: this is general function to check capability, may be used in future to test performance when one of keepalive method is forced

**Comment by hoopoepg** [comment]
@brminich more ideas?

File: src/tools/perf/lib/libperf.c (hunk 2 above)
Comment (yosefe) [reply src/tools/perf/lib/libperf.c:86]: i think it's ok to make sure all flags would be present here, I don't see a downside to adding it

**Review by brminich — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Pull request merged**